airflow/models.py

```diff
@@ -414,6 +416,16 @@ def process_file(self, filepath, only_if_updated=True, safe_mode=True):
                         self.bag_dag(dag, parent_dag=dag, root_dag=dag)
                         found_dags.append(dag)
                         found_dags += dag.subdags
+                        if isinstance(dag._schedule_interval, six.string_types):
```

Comment:
maybe put the check before we append the `dag` to `found_dags`?( if that's what we do for other dags with import errors)